packages/web/src/js/TextArea.ts

```diff
@@ -0,0 +1,47 @@
+import BaseComponent from './BaseComponent';
+import { enableToggleAutoloader } from './utils';
+import EventHandler from './dom/EventHandler';
+import { debounce } from './utils/Debounce';
+
+const NAME = 'textArea';
+const RESIZE_EVENT = 'resize';
+const DEBOUNCE_DELAY = 50;
+
+class TextArea extends BaseComponent {
+  textarea: HTMLTextAreaElement;
+
+  static get NAME() {
+    return NAME;
+  }
+
+  static get DATA_KEY() {
+    return `${this.NAME}`;
+  }
+
+  constructor(element: HTMLElement) {
+    super(element);
+    this.textarea = this.element.querySelector('textarea') as HTMLTextAreaElement;
+    this.adjustHeight = debounce(this.adjustHeight.bind(this), DEBOUNCE_DELAY);
+    this.adjustHeight();
+    this.addEventListeners();
+  }
+
+  addEventListeners() {
+    EventHandler.on(this.textarea, 'input', this.adjustHeight.bind(this));
```

Comment:
I think you can get rid of those binds by passing in the event and working with the event.target.

Reply:
Got rid of them.